packages/frontend-shared/src/gql-components/ChooseExternalEditor.vue

```diff
@@ -140,34 +124,28 @@ type Editor = ChooseExternalEditorFragment['localSettings']['availableEditors'][
 
 type EditorType = 'found' | 'custom'
 
-const selectedWellKnownEditor = ref<Editor | undefined>(
-  props.gql.localSettings.availableEditors.find((editor) => {
-    return editor.binary === props.gql.localSettings.preferences.preferredEditorBinary
-  }),
-)
+const selectedWellKnownEditor: Editor | undefined = props.gql.localSettings.availableEditors.find((editor) => {
+  return editor.binary === props.gql.localSettings.preferences.preferredEditorBinary
+})
 
 const customBinary = ref<string>(
-  selectedWellKnownEditor.value
+  selectedWellKnownEditor
     ? ''
     : props.gql.localSettings.preferences.preferredEditorBinary ?? '',
 )
 
-const editorToUse = ref<EditorType>(customBinary.value ? 'custom' : 'found')
+const editorToUse = ref<string>(
```

Comment:
Can you add back in `EditorType` here? Or remove the `EditorType` declarations since it's no longer used.